.github/workflows/update-internal-base-images.yml

```diff
@@ -24,8 +24,8 @@ jobs:
             git config --global user.name "auto dockerfiles update"
             echo "==== $(date): Starting pipenv setup... ===="
             python -m pip install --upgrade pip
-            pip install pipenv==2023.3.18
-            pipenv install
+            pip install pipenv==2024.4.0
+            pipenv install --python=$(which python)
```

Comment:
why is this flag needed

Reply:
Because otherwise it fails on [this](https://github.com/demisto/dockerfiles/actions/runs/12629465158/job/35187419878)